BrainPortal/app/models/cluster_task.rb

```diff
@@ -1618,6 +1616,11 @@ def unarchive_work_directory_from_userfile
       return false
     end
 
+    # todo 'file' command can be used to reliably determine actual archive type, Alpine etc ...
```

Comment:
Remove spurious comments (1619,1620). Yes we know we could make a more formal check, but there is no point in writing this down here.

Reply:
Do this